bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-message-list/page/chat-message/message-reactions/component.tsx

```
@@ -0,0 +1,33 @@
+    native: string;
+  }[];
+}
+
+const ChatMessageReactions: React.FC<ChatMessageReactionsProps> = (props) => {
+  const { reactions } = props;
+
+  return (
+    <Styled.ReactionsWrapper>
+      {reactions.map((emoji) => {
+        return (
+          <Styled.EmojiWrapper highlighted={false}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)